.github/workflows/build_and_deploy.yml

```diff
@@ -210,7 +221,7 @@ jobs:
       env:
         SLACK_CHANNEL: twd_tv_dev
         SLACK_USERNAME: CI Deployment
-        SLACK_TITLE: Deployment failure
+        SLACK_TITLE: Docker scan failure
```

Comment:
This relates to `wait_for_deploy_app` so it's relate to the deployment, why change it?